Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -1040,6 +1040,15 @@ public struct RenderNodeTranslator: SemanticVisitor {
                     return true
                 }
                 
+                // If the reference corresponds to an article, allow the topic
+                // to appear independently of the source language it belongs to.
+                // rdar://118461894
+                //
+                // TODO: Make article-only catalog pages language-less https://github.com/apple/swift-docc/issues/240
+                guard context.isSymbol(reference: reference) else {
```

Comment:
This guard-else statement confused me the first time I read it. Maybe it would be more obvious if you used the same syntax as the preceding comment: "If the reference corresponds to an article..."

```
if context.isArticle(reference: reference) {
  return true
}
```

but then you need an `isArticle` function instead.

Reply:
How is this meant to behave for other types of content? Is it only _articles_ that should always be included regardless of language or does this apply to tutorial content?

If this behavior is meant for all non-symbols then the comment should be updated to reflect that.

Reply:
Also, you don't need to leave radars links or issue numbers for bugs that are fixed. That information isn't all that useful in the future. If the change is so odd that it needs an explanation then that should be in a comment alone. 


The issue link in the TODO comment _may_ still be useful in the future for finding places in the code that needs an update when someone works on that issue. That said, that issue is that it's only for article-only catalogs so AFAIK we'd still need this check to not filter out language specific articles from catalogs that also contain reference documentation. If that's the case, then there's no action here for the person who works on #240 in the future, meaning that that TODO comment and link could also be removed.

Reply:
I'm updating the comment to
```
If the reference corresponds to any kind except Symbol 
(ex. Article, Collection, SampleCode...), allow the topic
to appear independently of the source language it belongs to.
```

I'm removing the radar and TODO comment but depending on how https://github.com/apple/swift-docc/issues/240 gets resolved that logic might not be needed anymore, for example, in the case that the solution is make any non-symbol page language-less.

Reply:
@patshaughnessy, this behaviour is meant for all non-symbol nodes, not just articles. My comment was a bit misleading. 

Another way to write it could be

```
if !context.isSymbol(reference: reference) {
    return true
}
```
However, using guard is a cleaner way to return early, similar to the approach in https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift#L1045

Reply:
Sounds good! Yep I was just confused earlier when the comment read one way, but the code read a different way. 100% agree `guard` statements are a great feature in Swift; makes a lot of code simpler. Thanks for updating the comment looks better now...